website/components/BlogIndex.tsx

```diff
@@ -23,13 +23,12 @@ export function BlogIndex({ more = "Read more" }) {
     .map((page) => {
       return (
         <div key={page.route} className="mb-10">
-          <Link href={page.route}>
-            <a
-              style={{ color: "inherit", textDecoration: "none" }}
-              className="block font-semibold mt-8 nx-text-2xl"
+          <Link
```

Comment:
See https://nextjs.org/docs/advanced-features/codemods#new-link for all the link related changes